config/routes.rb

```diff
@@ -35,9 +35,21 @@ def set_up_flipper
   namespace :partners do
     resource :dashboard, only: [:show]
     resource :help, only: [:show]
-    resources :requests, only: [:show, :new, :index, :create]
-    resources :individuals_requests, only: [:new, :create]
-    resources :family_requests, only: [:new, :create]
+    resources :requests, only: [:show, :new, :index, :create] do
+      collection do
+        post :validate
+      end
```

Comment:
```suggestion
      post :validate, on: :collection
```

A bit more concise when there's only one route being added.

Reply:
Done.